translations/en.yml

```diff
@@ -771,3 +771,16 @@ run_usage: |
   your code
   `​`​`
 run_output: Run Output
+
+# Alert Channel
+member: Member
+alert_channel: Alert Channel
+alert_channel_set: The alert channel has been changed.
+alert_channel_get: Current Alert Channel
+
+## Channel hopping
```

Comment:
Please remove unnecessary hashtag

Reply:
I can't commit any new changed to this PR

Reply:
fixed in experimental